src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -963,6 +940,38 @@ private static void updateVelocity(Entity e, double scale)
             }
             if (e instanceof LivingEntity) ((LivingEntity) e).setHealth(health);
         });
+
+        put("mayfly", (e, v) -> {
+            if (v == null) {
+                throw new InternalExpressionException("'mayfly' requires parameters");
+            }
```

Comment:
This is handled automatically, no need for check

Reply:
nvm abt this

Reply:
It actually is, there's a try-catch for NPE outside in the method that calls this.

Reply:
Well do mind then, never mind me telling you not to mind about this.

Reply:
(also I thought I remembered seeing a try/catch there, hence why I sent the message, but then seeing this comment https://github.com/gnembon/fabric-carpet/pull/1160#discussion_r744160669 confused me a bit cos I didn't understand the context)